community-operators/grafana-operator/grafana-operator.v1.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,262 @@
+    mediatype: image/svg+xml
+  install:
+    spec:
+      deployments:
+      - name: grafana-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: grafana-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: grafana-operator
+            spec:
+              containers:
+              - args:
+                - --grafana-image=quay.io/openshift/origin-grafana
+                - --grafana-image-tag=4.2
+                - --openshift
```

Comment:
Are operators under `community-operators` only offered on OpenShift 4? This flag makes the operator incompatible with vanilla Kubernetes by allowing it to create a Route (instead of Ingress) and prepare the use of oauth-proxy.

Is this an intended use for `community-operators` ?